src/adapters/yieldbot.js

```diff
@@ -33,8 +33,7 @@ var YieldbotAdapter = function YieldbotAdapter() {
      * @private
      */
     buildCreative: function (slot, size) {
-      return '<script type="text/javascript" src="//cdn.yldbt.com/js/yieldbot.intent.js"></script>' +
-        '<script type="text/javascript">var ybotq = ybotq || [];' +
+      return '<script type="text/javascript">var ybotq = ybotq || [];' +
```

Comment:
Do you have a test for this? This would work in principle if, of course, the `yieldbot` object is in the frame in which the creative script is evaluated or accessible in the parent frame.

If we look at [Prebid, step-2-add-a-creative](http://prebid.org/adops/step-by-step.html#step-2-add-a-creative), the creative will be served into a "friendly" iframe; so, the `yieldbot` object needs to be available.

The Prebid.js creative looks like this (*from the link above*):
```
<script>
var w = window;
for (i = 0; i < 10; i++) {
  w = w.parent;
  if (w.pbjs) {
    try {
      w.pbjs.renderAd(document, '%%PATTERN:hb_adid%%');
      break;
    } catch (e) {
      continue;
    }
  }
}
</script>
```
Notice `w = w.parent;`. Since the DFP setup **requires** a friendly iframe for rendering, the child render frame can access the parent window object to call `pbjs.renderAd()`.

So, this change *should* ™️ work.